modules/__tests__/matchPattern-test.js

```diff
@@ -41,4 +41,8 @@ describe('matchPattern', function () {
     assertMatch('/**/*.jpg', '/files/path/to/file.jpg', '', [ 'splat', 'splat' ], [ 'files/path/to', 'file' ])
   })
 
+  it('works with patterns that match built-in names', function () {
```

Comment:
Not here – `matchPattern` prepends a `/` to all patterns to match the pathname.

The failing test case is the `getParamsName` one, which currently would fail on all browsers.